docs/reference/release-notes/7.0.0-rc1.asciidoc

```diff
@@ -1,6 +1,203 @@
 [[release-notes-7.0.0-rc1]]
 == {es} version 7.0.0-rc1
 
-Also see <<breaking-changes-7.0,Breaking changes in 7.0>>.
+coming[7.0.0-rc1]
+
+Also see <<breaking-changes-7.0.0-rc1,Breaking changes in 7.0.0-rc1>>.
```

Comment:
we should drop "-rc1" since we don't generate a list of breaking changes that are specific to RCs.